cluster-scope/base/routes/elastic-operator/route.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: route.openshift.io/v1
+kind: Route
```

Comment:
I don't like routes to be created within the cluster-scope application, they are really namespace specific are are just cluttering things in here. WDYT @HumairAK , I'd suggest to piggy back on the `Subscription` resource in this case and just deploy it together with the operator.

Reply:
@tumido This route exposes the entire elastic-search instance - there was another option which is to connect Logging to Kafka Bus ... if you can explain / point to Kafka ... we can drop this route and forward the logs to Kafka

Reply:
@tumido for now I disabled (comment) the deployment of this route as part of the main kustomization.yaml file ... so let's see that this deployment works and continue from there

Reply:
I don't want to clutter the `cluster-scope` application with a namespaced resources that much. Can you please either remove it or put it within a separate application in the repository root?

Reply:
removed the route all together for now ... let's start to see logs and that I will work with Bruno on how to expose them ... I agree that this potentially can be app level.